packages/matchbox/src/components/Banner/Banner.js

```diff
@@ -1,144 +1,151 @@
-    </div>
+    <Box
+      position="relative"
+      // IconSection do not render under the first breakpoint
+      display={['none', 'none', 'flex']}
+      alignItems="center"
+      justifyContent="center"
+      width="3rem"
+      height="3rem"
+      mr="500"
+    >
+      <Box
+        position="absolute"
+        top="0"
+        left="0"
+        width="3rem"
+        height="3rem"
+        borderRadius="circle"
+        bg={statusIcon.bg}
+      />
+      <Box position="relative" color={statusIcon.fill}>
+        <Icon size={30} label={statusIcon.iconLabel} />
+      </Box>
+    </Box>
   );
-};
+}
 
-class Banner extends Component {
-  static displayName = 'Banner';
-
-  static propTypes = {
-    /**
-     * The type of banner. 'default' | 'success' | 'warning' | 'danger' | 'info'
-     */
-    status: PropTypes.oneOf(['default', 'success', 'warning', 'danger', 'info']),
-
-    /**
-     * The banner's title
-     */
-    title: PropTypes.string,
-
-    /**
-     * Callback when dismiss button is clicked. Button hidden without callback.
-     */
-    onDismiss: PropTypes.func,
-
-    /**
-     * Action that build a button. Most button props will work in here.
-     * e.g. { content: 'button label', onClick: callback() }
-     */
-    action: PropTypes.shape({ content: PropTypes.string.isRequired }),
-
-    /**
-     * List of actions that build buttons. Most button props will work in here.
-     * Overrides `action`
-     */
-    actions: PropTypes.arrayOf(PropTypes.shape({
-      content: PropTypes.string.isRequired
-    })),
-
-    /**
-     * Banner Content
-     */
-    children: PropTypes.node
-  };
-
-  static defaultProps = {
-    status: 'default'
-  };
-
-  render() {
-    const {
-      children,
-      title,
-      status,
-      action,
-      actions,
-      onDismiss,
-      ...rest
-    } = this.props;
-
-    const titleMarkup = title
-      ? <h5 className={styles.Title}>{title}</h5>
-      : null;
-
-    let actionMarkup = action
-      ? <div className={styles.Actions}>{buttonFrom(action, actionOverrides)}</div>
-      : null;
+const StyledContainer = styled(Box)`
+  ${container}
+  ${margin}
+  ${childLinks}
+`;
+
+const StyledDismiss = styled(Box)`
+  ${buttonReset}
+  ${dismissBase}
+  ${dismissColor}
+`;
+
+function Banner(props) {
+  const { children, title, status, action, actions, onDismiss, ...rest } = props;
+
+  const titleMarkup = title ? (
+    // IconSection does not render under first breakpoint
+    // `pt` here is used to vertically align the title with the icon
+    <Box pt={['0', '0', '200']} mb="200">
+      <Text fontSize="500" lineHeight="500" as="h5">
+        {title}
+      </Text>
+    </Box>
+  ) : null;
+
+  const actionMarkup = React.useMemo(() => {
+    let result = action ? <div>{buttonFrom(action)}</div> : null;
 
     if (actions) {
-      actionMarkup = (
-        <div className={styles.Actions}>
-          {actions.map((action, i) => buttonFrom(action, actionOverrides, i))}
```

Comment:
We don't need `actionOverrides` anymore?

Reply:
The new banners use both outlined / non outlined, normal sized buttons, I figured it would be best to have the flexibility here instead of specifying which buttons should be outlined / non outlined